modules/util/portal-tools-rest-builder/src/main/java/com/liferay/portal/tools/rest/builder/internal/freemarker/tool/java/parser/util/OpenAPIParserUtil.java

```diff
@@ -559,7 +559,7 @@ private static void _addExternalReferences(
 				String reference = _getReference(currentSchema);
 
 				if (reference != null) {
-					if (reference.contains("#/components/schemas/")) {
+					if (reference.startsWith("#/components/schemas/")) {
```

Comment:
Could this be a breaking change?

Reply:
It could. We will get back to this in the future.